about.html

```diff
@@ -12,62 +12,91 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
+  <div class="container">
+    <header>
+      <img class="logo" src="C:\Users\Melissa\lambda\Sprint-Challenge--User-Interface\img\lambda-black.png" alt="Lambda School Logo.">
```

Comment:
you need to use a relative address, relative to the location of the file your referencing from. this link is broken when its online.. you want 'img/lambda-black.png'